setup.cfg

```
@@ -82,7 +82,7 @@ plugins =
 
 [coverage:report]
 precision = 2
-fail_under = 97.82
+fail_under = 97.87
```

Yey! :tada:

---

Let's try to increase it! :smiling_imp: 
```suggestion
fail_under = 97.92
```